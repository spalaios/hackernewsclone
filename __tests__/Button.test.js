import React from 'react';
import renderer from 'react-test-renderer';
import MoreButton from '../src/components/presentation/Button/Button';
// import MoreButton from '../../src/components/presentation/Button/Button';

test('Testing the button component', () => {
  const component = renderer.create(
    <MoreButton handleClick={() => 'clicked more btn'} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});