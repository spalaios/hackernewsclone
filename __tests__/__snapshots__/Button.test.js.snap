// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the button component 1`] = `
<button
  className="more-button mt-3 mb-2 cursor"
  onClick={[Function]}
  type="button"
>
  More
</button>
`;
